march_rqt_gait_generator/src/march_rqt_gait_generator/gait_generator.py

```diff
@@ -368,7 +368,7 @@ def update_gait_duration(self, duration):
     def import_gait(self):
         file_name, f = QFileDialog.getOpenFileName(self._widget,
                                                    "Open Image",
-                                                   rospkg.RosPack().get_path('march_gait_files'),
+                                                   os.getenv("HOME") + "/march_ws/src/gait-files/march_gait_files",
```

Comment:
You make an assumption about the name of the workspace, which is completely up to the user

Reply:
Yes, but there was no elegant way to do this; if your ws is named differently you will be redirected to "/home/< user>/.ros" which is not a big problem.